app/assets/stylesheets/base/_logos.scss

```diff
@@ -28,4 +37,26 @@ $logo-height-desktop: 50px;
       @include responsive-styles(height, rem-calc(22), rem-calc(36), rem-calc(36));
     }
   }
+  &--unep {
+    background-image: image-url('icons/unep-logo.svg');
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: $logo-unep-width-mobile; height: $logo-height-mobile;
+
+    display: block;
+    margin-right: auto;
+
+    @include breakpoint($small) { width: $logo-unep-width-tablet; height: $logo-height-tablet; }
+    @include breakpoint($medium) { width: $logo-unep-width-desktop; height: $logo-height-desktop; }
+
+    .section--hero & {
+      @include responsive-styles(width, rem-calc(50), rem-calc(75), rem-calc(75));
+      @include responsive-styles(height, rem-calc(34), rem-calc(52), rem-calc(52));
+    }
+
+    .footer & {
+      @include responsive-styles(width, rem-calc(32), rem-calc(55), rem-calc(55));
+      @include responsive-styles(height, rem-calc(22), rem-calc(36), rem-calc(36));
+    }
+  }
```

Comment:
Line 20 to 38 can probably end up in a mixin (with parameters to account for the different values in rem-calc (in responsive-styles)) that we can use in logo--unep and logo--wcmc. This would save repeating the code.